packages/ifpack2/src/Ifpack2_Factory.cpp

```diff
@@ -28,11 +28,7 @@ bool supportsUnsymmetric(const std::string& prec_type) {
       prec_type == "RBILUK" ||
       prec_type == "ILUT" ||
       prec_type == "SCHWARZ" ||
-      prec_type == "KRYLOV"
-#ifdef HAVE_IFPACK_HYPRE
```

Comment:
I suspect this is a bug and should instead be `HAVE_IFPACK2_HYPRE`.

Reply:
@sebrowne I confirmed that this is a bug.  It's been fixed by #14301, which is why there's now a conflict.